Blocking problem with the latest build (post 0.99.6) on Windows

There is a blocking problem with the latest build (post 0.99.6) on Windows.
The `docs-client` was always giving me headaches (yarn frequent timeout errors, yarn proxy issues), but now it just stopped working due to line breaks incompatibility issue.
The build fails on the following command:
```
…\.gradle>nodejs\node-v12.17.0-win-x64\node.exe npm\npm-v6.14.5\node_modules\npm\bin\npm-cli.js run lint
```
For every single file it processes it gives an error for each line, like below:
```
…\armeria\docs-client\.eslintrc.js
  1:62  error  Delete `␍`  prettier/prettier
  2:1   error  Delete `␍`  prettier/prettier
  ...
```
I could only pass it when I manually forced it to "fix" the line breaks with `--fix` option, but that, of course, modified all the processed files.
Please consider a fix for the Windows builds.

There is also one other problem with Windows build. During the build, line breaks always change for the following files:
```
examples/grpc-kotlin/gen-src/main/java/example/armeria/grpc/kotlin/Hello.java
examples/grpc-kotlin/gen-src/main/grpc/example/armeria/grpc/kotlin/HelloServiceGrpc.java
```
This problem existed even before 0.99.6 but was not blocking, although was quite annoying.

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -27,7 +27,7 @@ init:
 
 build_script:
   - cmd: 'gradlew.bat --version'
-  - cmd: 'gradlew.bat %GRADLE_CLI_OPTS% -PnoLint -PnoWeb :core:test :grpc:test :thrift:test :it:server:test'
+  - cmd: 'gradlew.bat %GRADLE_CLI_OPTS% lint :core:test :grpc:test :thrift:test :it:server:test'
   - sh: './.appveyor.build.sh "$PROFILE"'
 
 test: off

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -26,12 +26,8 @@ init:
   - sh: 'git config --global core.autocrlf input'
 
 build_script:
-  - cmd: 'git rm -fr .'
-  - cmd: 'git reset --hard HEAD'
   - cmd: 'gradlew.bat --version'
-  - cmd: 'gradlew.bat %GRADLE_CLI_OPTS% lint :core:test :grpc:test :thrift:test :it:server:test'
-  - sh: 'git rm -fr .'
-  - sh: 'git reset --hard HEAD'
+  - cmd: 'gradlew.bat %GRADLE_CLI_OPTS% -PnoLint -PnoWeb :core:test :grpc:test :thrift:test :it:server:test'
   - sh: './.appveyor.build.sh "$PROFILE"'
 
 test: off

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -36,7 +36,7 @@ on_failure:
   - sh: './.appveyor.report.sh "$PROFILE"'
 
 cache:
-  - "%USERPROFILE%\\AppData\\Local\\Yarn\\cache"
+  - "%APPDATA%\\npm-cache"
   - "%USERPROFILE%\\.gradle\\wrapper\\dists"
   - "%USERPROFILE%\\.gradle\\caches\\jars-3"
   - "%USERPROFILE%\\.gradle\\caches\\modules-2"

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -26,8 +26,12 @@ init:
   - sh: 'git config --global core.autocrlf input'
 
 build_script:
+  - cmd: 'git rm -fr .'
+  - cmd: 'git reset --hard HEAD'
   - cmd: 'gradlew.bat --version'
   - cmd: 'gradlew.bat %GRADLE_CLI_OPTS% lint :core:test :grpc:test :thrift:test :it:server:test'
+  - sh: 'git rm -fr .'
+  - sh: 'git reset --hard HEAD'
   - sh: './.appveyor.build.sh "$PROFILE"'
 
 test: off

--- a/.gitattributes
+++ b/.gitattributes
@@ -20,10 +20,17 @@
 *.m4a binary
 *.m4v binary
 *.mp4 binary
+*.otf binary
 *.p12 binary
 *.pdf binary
 *.pem binary
 *.pkcs12 binary
 *.png binary
+*.ttc binary
+*.ttf binary
+*.webm binary
+*.webp binary
+*.woff binary
+*.woff2 binary
 *.xz binary
 *.zip binary

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,8 +1,29 @@
+* text=auto eol=lf
+
 *.bat text eol=crlf
+*.cmd text eol=crlf
+
+*.ai binary
+*.binarypb binary
 *.br binary
+*.class binary
+*.crt binary
+*.eot binary
+*.exe binary
+*.gif binary
 *.gz binary
+*.ico binary
+*.jar binary
 *.jks binary
+*.jpg binary
+*.jpeg binary
+*.m4a binary
+*.m4v binary
+*.mp4 binary
+*.p12 binary
+*.pdf binary
+*.pem binary
 *.pkcs12 binary
-
-/gradlew text eol=lf
-
+*.png binary
+*.xz binary
+*.zip binary

